packages/template-retail-react-app/app/pages/checkout/util/checkout-context.js

```diff
@@ -311,7 +313,12 @@ export const CheckoutProvider = ({children}) => {
                 try {
                     await basket.createOrder()
                 } catch (error) {
-                    mergeState({globalError: error.message})
+                    // TODO: OCAPI implements localized error messages for some error types
+                    const message = formatMessage({
+                        id: 'checkout_confirmation.message.generic_error',
```

Comment:
```suggestion
                        id: 'checkout.message.generic_error',
```
Minor: when the error message shows up, we're still in checkout and has not reached the confirmation page yet (where the order submission is successful)